main.ledger

```diff
@@ -1191,12 +1346,17 @@
 2015/09/30 Wells Fargo
     Expenses:Services:Bank:Wells Fargo                   $6.00
     Assets:Bank:Savings
```

Comment:
Wells Fargo service charge. Since it's such a small expense, receipt isn't
necessary from our end (though if you'd like, I can get it).

On Sat, Feb 20, 2016 at 11:46 AM, JESSICA <notifications@github.com> wrote:

> In main.ledger
> <https://github.com/hackclub/finances/pull/8#discussion_r53556068>:
>
> > @@ -1191,12 +1346,17 @@
> >  2015/09/30 Wells Fargo
> >      Expenses:Services:Bank:Wells Fargo                   $6.00
> >      Assets:Bank:Savings
>
> Wait what happened here; where's the receipt for this
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackclub/finances/pull/8/files#r53556068>.
>



-- 
Zach Latta
https://zachlatta.com <http://zachlatta.com>